src/front/js/component/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "../../styles/Header.css";
```

Comment:
I saw also that you named some files in this capitalized way, but I think it's better if we keep them all as lowercase, okay?

I know that maybe flux uses camelCase with their files, but... let's just keep that there 🐫